selfdrive/controls/controlsd.py

```diff
@@ -387,8 +357,7 @@ def data_send(sm, pm, CS, CI, CP, VM, state, events, actuators, v_cruise_kph, rk
     "alertBlinkingRate": AM.alert_rate,
     "alertType": AM.alert_type,
     "alertSound": AM.audible_alert,
-    "awarenessStatus": max(driver_status.awareness, -0.1) if isEnabled(state) else 1.0,
-    "driverMonitoringOn": bool(driver_status.face_detected),
+    "driverMonitoringOn": bool(sm['monitorState'].driverState.faceDetected),
```

Comment:
don't need the bool wrapper anymore. This is already a bool from the capnp structure.